src/common/constants.ts

```diff
@@ -102,7 +102,7 @@ const Constants = {
     Moderator: 'Moderator',
     Suspended: 'Suspended',
   },
-  ShouldShowAuthorOnRegister: true,
+  ShouldShowAuthorOnRegister: false,
```

Comment:
Was this on purpose?

Reply:
i don't think this constant should live here since its just a tweak for how the internal of one plugin/service should run. is it possible to have it live in either of those files?

Reply:
we can move it for now, this idea sparked #729 so eventually it will be moved there